configure.ac

```diff
@@ -545,6 +545,7 @@ AC_CONFIG_FILES([Makefile src/Makefile
                  src/plugins/Makefile
                  src/dirwatch/Makefile
                  src/libusermode/Makefile
+                 src/librepl/Makefile
```

Comment:
This new library should be conditionally enabled (`--enable-repl`) and be off by default.